src/logger/telemetry-transport.ts

```diff
@@ -60,7 +60,11 @@ export class TelemetryTransport extends winston.Transport {
                         }
                     }
                 }
-                this.reporter.sendTelemetryEvent(message, properties, measures);
+                if (level === "info") {
```

Comment:
Could you please set "info" "error" and "warn" as an enum like "LogLevel" and replace the hard-coded string in the files?

Reply:
updated.